qcodes/dataset/guids.py

```diff
@@ -99,14 +99,13 @@ def set_guid_location_code() -> None:
         return
 
     cfg['GUID_components']['location'] = location
-    cfg.save_to_home()
```

Comment:
This one should really call save_to_home as it is meant to run interactively when the user sets the config

Reply:
yes